src/components/PastTripForm.js

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import ButtonAdd from './Button';
+
+export default function PastTripNotes({ onHandleNewNote }) {
+  const [textInput, setTextInput] = useState('');
+
+  return (
+    <>
+      <NotesForm
+        aria-labelledby="notes-form"
```

Comment:
I don't think that it is a good practice to use a Button as a reference to label the form; maybe use a heading you hide via a sr-only style.